### DISCUSSION TITLE
virtual threads

### DISCUSSION THREAD

**Discussion by hustchaya** [async-profiler/async-profiler#949] [Q&A] [answered]
Could you please share your core approach if async-profiler supports virtual threads? Additionally, do you have any scheduling availability for this on your end

**Comment by apangin** [comment, marked as answer]
Thank you for your interest in this feature.
The question has already been answered in #802.

**Reply by hustchaya** [reply]
I have actually read through this issue before. This time, I was hoping to consult you to see if you have any preliminary thoughts or analysis on virtual threads
